lux/vislib/altair/Histogram.py

```diff
@@ -53,7 +53,7 @@ def initialize_chart(self):
 
         # Default when bin too small
         if markbar < (x_range / 24):
-            markbar = (x_max - x_min) / 12
+            markbar = abs(x_max - x_min) / 12
```

Comment:
Would there ever be a case such that calling `abs` is necessary? By construction, shouldn't `x_max` be greater than or equal to `x_min`?

Reply:
I agree, I added this as a final check for integer overflow so that at least our code doesn't error and it just means the user inputs are too extreme (positive or negative) to handle. @dorisjlee should we even worry about this?

Reply:
I think xmax will always be larger than xmin.